qa/comparitor.py

```diff
@@ -0,0 +1,180 @@
+import csv
+import requests
+from datetime import datetime, timedelta
+
+from dictdiffer import diff
+
+from qa.metrics import METRICS
+
+class MetricDiff(object):
+    state = None
+    county = None
+    fips = None
+    date = None
+    metric_name = None
+
+    snapshot1 = None
+    snapshot2 = None
+
+    value1 = None
+    value2 = None
+
+    threshold = None
+    difference = None
+
+    def __init__(self, state, county, fips, date, metric_name, snapshot1, snapshot2, value1, value2, difference, threshold):
+        self.state  = state
+        self.county = county
+        self.fips = fips
+        self.date = date
+        self.metric_name = metric_name
+        self.snapshot1 = snapshot1
+        self.snapshot2 = snapshot2
+        self.value1 = value1
+        self.value2 = value2
+        self.threshold = threshold
+        self.difference = difference
+
+
+class Comparitor(object): 
+    snapshot1 = None
```

Comment:
small note, but this defines them as class variables, not instance variables - could make this a dataclass too!